### PR TITLE
media/utc: Restore volume after test

### PR DESCRIPTION
**Pull request by junmin-kim**
Before testing setVolume, save previous volume and restore after test

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [9a29802c3988912ac40e82aae41c8f7c658c3afd] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [9a29802c3988912ac40e82aae41c8f7c658c3afd] - Code Rule Check (C++) OK.

File: apps/examples/testcase/ta_tc/media/utc/utc_media_mediarecorder.cpp [hunk 1]
```diff
@@ -105,26 +105,39 @@ static void utc_media_MediaRecorder_setDataSource_n(void)
...
 	TC_ASSERT_EQ("utc_media_mediarecorder_setVolume", mr.setVolume(10), RECORDER_ERROR_NOT_ALIVE);
 	mr.create();
 	mr.setDataSource(std::move(dataSource));
 	mr.prepare();
-	TC_ASSERT_EQ("utc_media_mediarecorder_setVolume", mr.setVolume(11), RECORDER_OK);
+	mr.getVolume(&prev);
+	TC_ASSERT_EQ_CLEANUP("utc_media_mediarecorder_setVolume", mr.setVolume(11), RECORDER_OK, goto cleanup);
+
+cleanup:
+	mr.setVolume(prev);
```
Comment (YiHyunJin) [inline apps/examples/testcase/ta_tc/media/utc/utc_media_mediarecorder.cpp]: @junmin-kim Does it need to do set prev in situations where it does not set?

Comment (junmin-kim) [reply apps/examples/testcase/ta_tc/media/utc/utc_media_mediarecorder.cpp]: This test try to set volume as 11, but set 10(max value).
So it is needed to restore volume

**Pull request merged**